src/Main.java

```diff
@@ -2487,13 +2683,19 @@ else if (region == 0x14) {
             }
             else if (region == 0x19) {
                 if (location == 0) {
+				// This was not originally checking for Boots, which is required for HARD logic. I fixed this issue.
                     return canSwim(inventory)
-                            && inventory.contains(Items.GARLIC)
-                            && (difficulty >= Difficulty.HARD || canSuperGP(inventory));
+                        && inventory.contains(Items.GARLIC)
+                        && ((difficulty >= Difficulty.HARD && inventory.contains(Items.JUMP_BOOTS)) || canSuperGP(inventory));
                 }
                 else {
+				/*
+				* Added MINOR GLITCHES logic for this check, which is Spiders Side Room - Lower Right.
+				* Break the bottom next to this check, then do a Reverse High Walljump.
+				*/
                     return canSwim(inventory)
-                            && inventory.contains(Items.GARLIC);
+                        && (inventory.contains(Items.GARLIC)
+						|| (difficulty >= Difficulty.S_HARD && inventory.constructor(Items.JUMP_BOOTS));
```

Comment:
`constructor` should be `contains`

Reply:
Looks like I must have tabbed too early on this one so it gave the wrong function. Fixed